src/main/java/seedu/address/logic/parser/specialisation/SpecialisationSetCommandParser.java

```diff
@@ -8,6 +8,8 @@
 import seedu.address.model.programmes.specialisations.cs.AlgorithmsAndTheorySpecialisation;
 
 public class SpecialisationSetCommandParser implements Parser<SpecialisationSetCommand> {
+
+    public static final String INVALID_SPECIALISATION = "Invalid Specialisation";
```

Comment:
Could add format string for the given specialisation input (e.g. "Invalid specialisation: %1$s")